docs-chef-io/content/inspec/resources/podman.md

```diff
@@ -0,0 +1,176 @@
++++
+title = "podman resource"
+draft = false
+gh_repo = "inspec"
+platform = "unix"
+
+[menu]
+  [menu.inspec]
+    title = "podman"
+    identifier = "inspec/resources/os/podman.md podman resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `podman` Chef InSpec audit resource to test the configuration data for the Podman resources.
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
```

Comment:
here either 'describes' or 'allow' would suffice. I have reatained 'allow' on such occurance throughout this resource writeup. Let me know your thoughts.